buildout.cfg

```diff
@@ -5,6 +5,7 @@ develop = .
 parts =
     interpreter
     test
+    omelette
```

Comment:
Please remove omelette and references to it, we don't use that in zopefoundation projects.